### ISSUE TITLE
Guard UI - Sword and Armor Not Shown in Inv

### ISSUE THREAD

**Issue by NerdyMindNYC** [TangoTek/TekTopia-Community#243]
**Describe the bug**
Can't keep an eye on the guard's sword durability or their armor.  

**To Reproduce**
1. Have Guard
2. Equip with gear and weapon
3. Click guard
4. See nothing

**Expected behavior**
Ability to see, at least, their weapon so we can replace it asap if about to break.

**Screenshots**
![GUARD UI](https://user-images.githubusercontent.com/51106650/58748030-0e598800-8441-11e9-8c9e-e7b48447b1aa.png)

** Server/Location **
What server were you playing on? Main(Survival) or Dev (Creative)? What If 2
What are the world coordinates of your town hall marker? 6850 7500

**Additional context**
The armor I understand if it functions like minecraft ui, but sword is a tool like any other villager and would be helpful if we can view the durability of said item.

**Comment by JohnyDL** [comment]
Duplicate of #214 very low priority

**Comment by NerdyMindNYC** [comment]
swear i searched "guard" first but thank you! ♡

**Comment by JohnyDL** [comment]
Yeah didn't for me either but I commented on it and searched for what I remember typing XD

**Comment by LadyGwynn** [comment]
I agree. Tango responded to this issue in #214 saying that he would have to make a new armor slot UI for the guards to fix this, but that doesn't make sense for the sword, as it doesn't go into a special inventory slot, it is used from the hotbar, just like picks, axes, and hoes.
